.github/workflows/tauri-release.yml

```
@@ -1,10 +1,7 @@
 name: Tauri Publish
 
 # This will trigger the action on each push to the `release` branch.
-on:
-  push:
-    branches:
-      - main
+on: [push]
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)